types/stripe-v3/index.d.ts

```diff
@@ -1024,6 +1025,7 @@ declare namespace stripe {
             paymentRequest?: paymentRequest.StripePaymentRequest | undefined;
             supportedCountries?: string[] | undefined;
             disabled?: boolean | undefined;
+            disableLink?: boolean | undefined;
```

Comment:
IMHO: there is no need to add the explicit `undefined` type, but it respects the consistency of the code.